src/shell/seats.rs

```diff
@@ -172,7 +172,7 @@ pub fn create_seat(
     output: &Output,
     config: &Config,
     name: String,
-) -> Seat<State> {
+) -> (Seat<State>, KeyboardHandle<State>) {
```

Comment:
Lets not change this api. We can simply call `initial_seat.get_keyboard()` afterwards to get the keyboard.